src/map/ai/controllers/mob_controller.cpp

```diff
@@ -1149,18 +1141,30 @@ bool CMobController::Cast(uint16 targid, SpellID spellid)
 bool CMobController::CanMoveForward(float currentDistance)
 {
     TracyZoneScoped;
-    if (PMob->m_Behaviour & BEHAVIOUR_STANDBACK && currentDistance < 20)
+
+    uint16 standbackRange = 20;
```

Comment:
Out of scope for this PR, but does this ever change?

Reply:
I set it up this way to mimic what Eden is doing in this block; they have a mobmod for this and we dont, so I left it hard-coded

Reply:
we probably should follow up with adding that mobmod - I am fairly sure there are a few mobs on retail that disobey the standard rules and that's also a fun customization point too. 

Note: we could also convert the one that handles if they standback at all, and  if we don't want them to, just set it zero, but that both creates a barrier to them eventually getting back in sync with us and is potentially a surprise bug situation later in existing code.